src/Expression.php

```diff
@@ -383,6 +383,26 @@ public function getDebugQuery()
         return $d.' <span style="color:gray">[' . implode(', ', $pp) . ']</span>';
     }
 
+    function __debugInfo()
+    {
+
+        $arr = [
+            'R'=>false,
+            'template'=>$this->template,
+            'params'=>$this->params,
+            'connection'=>$this->connection,
+            'args'=>$this->args,
+        ];
+
+        try {
+            $arr['R'] = $this->render();
+        } catch (\Exception $e) {
```

Comment:
even better would be:
```
catch (Exception $e) {
    $arr['R'] = $e->getMessage();
    $arr['R-params'] = $e->getParams();
} catch (\Exception $e) {
    $arr['R'] = $e->getMessage();
}
```
to show all params too in case this is dsql exception :)